apps/oc_chef_wm/src/oc_chef_wm_base.erl

```diff
@@ -812,6 +822,32 @@ api_info_header_value(#base_state{}=State) ->
 add_api_info_header(Req, State) ->
     wrq:set_resp_header("X-Ops-API-Info", api_info_header_value(State), Req).
 
+%% @doc Validate the requested X-Ops-Server-API-Version value and populate base_state,
+%% or fail with a 406 if it's not supported. Replies with base_state on success.
+server_api_version(_Req, State, undefined) ->
+    State#base_state{server_api_version = ?API_MIN_VER};
+server_api_version(Req, State, RequestedVersion) ->
```

Comment:
Is there a reason to pass a 3rd arg into this function since it's just being pulled out of the Req anyway?